plugins/reporters/spdx/src/main/kotlin/Extensions.kt

```diff
@@ -154,6 +156,26 @@ internal fun Package.toSpdxPackage(
         .filterExcluded()
         .filter(LicenseView.ONLY_DETECTED)
 
+    val declaredPackageLicenses = declaredLicensesProcessed.toSpdxDeclaredLicense()
+    val foundPackageLicenses = declaredPackageLicenses.takeUnless {
+        it == SpdxConstants.NOASSERTION && type == SpdxPackageType.VCS_PACKAGE
+    } ?: run {
+        // If there are of declared licenses and the package type is a VCS, which never have metadata, fall back to
+        // determine the package's license from what is detected in root license files.
+        val patterns = LicenseFilePatterns.getInstance()
```

Comment:
Does this instance use the configured patterns?

Reply:
Yes.